Arrangement_on_surface_2/test/Arrangement_on_surface_2/cgal_test.cmake

```diff
@@ -659,6 +659,17 @@ function(test_construction_segments)
   compile_and_run_with_flags( test_construction segments "${flags}")
 endfunction()
 
+#---------------------------------------------------------------------#
+# construction with segments, without Union Find
+#---------------------------------------------------------------------#
+function(test_construction_segments_no_union_find)
+  set(nt ${CGAL_GMPQ_NT})
+  set(kernel ${CARTESIAN_KERNEL})
+  set(geom_traits ${SEGMENT_GEOM_TRAITS})
+  set(flags "-DTEST_NT=${nt} -DTEST_KERNEL=${kernel} -DTEST_GEOM_TRAITS=${geom_traits} -DCGAL_ARRANGEMENT_DISABLE_UNION_SET_CCB")
+  compile_and_run_with_flags( test_construction segments_no_union_find "${flags}")
+endfunction()
```

Comment:
1. You add a function, but that does not add any test if you do not all it.
2. The Arr_2 test suite is complex, and it has also a non-CTest version, in `cgal_test_with_cmake`.